{"name": "test_a", "status": "failed", "statusDetails": {"message": "AssertionError: assert 2 == 3", "trace": "self = <Test_App3.Test_ABC object at 0x07A0A670>, need_data = 2\n\n    def test_a(self,need_data):\n        print(\"------->test_a\")\n>       assert need_data == 3 # 拿到返回值做一次断言\nE       assert 2 == 3\n\nTest_App3.py:11: AssertionError"}, "start": 1602662700650, "stop": 1602662700650, "uuid": "f7e401dc-850f-49be-953d-bb9fb050ea28", "historyId": "733a399fe5e6fcfad4eb10bd1acdda47", "testCaseId": "716505b0eb184796ac5f3c6ae860d9a6", "fullName": "clsp.Test_App3.Test_ABC#test_a", "labels": [{"name": "parentSuite", "value": "clsp"}, {"name": "suite", "value": "Test_App3"}, {"name": "subSuite", "value": "Test_ABC"}, {"name": "host", "value": "LAPTOP-LFNOOUSC"}, {"name": "thread", "value": "11208-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "clsp.Test_App3"}]}